src/cascadia/TerminalApp/CascadiaSettingsSerialization.cpp

```diff
@@ -148,6 +241,101 @@ void CascadiaSettings::SaveAll() const
     _WriteSettings(serializedString);
 }
 
+// Method Description:
+// - Finds all the dynamic profiles we've generated that _don't_ exist in the
+//   user's settings. Generates a minimal blob of json for them, and inserts
+//   them into the user's settings at the end of the list of profiles.
+// - Does not reformat the user's settings file.
+// - Does not write the file! Only modifies in-place the _userSettingsString
+//   member. Callers should make sure to call
+//   _WriteSettings(_userSettingsString) to make sure to persist these changes!
+// - Assumes that the `profiles` object is at an indentation of 4 spaces, and
+//   therefore each profile should be indented 8 spaces. If the user's settings
+//   have a different indentation, we'll still insert valid json, it'll just be
+//   indented incorrectly.
+// Arguments:
+// - <none>
+// Return Value:
+// - true iff we've made changes to the _userSettingsString that should be persisted.
+bool CascadiaSettings::_AppendDynamicProfilesToUserSettings()

```

Comment:
offline disc: keep this function, but make the entry emitted into the json file _only the guid,source and name_